t/mojolicious/secret/lite-create.t

```diff
@@ -0,0 +1,16 @@
+use Mojo::Base -strict;
+
+use Mojo::File qw(tempfile);
+use Test::Mojo;
+use Test::More;
+use Mojolicious::Lite;
+
+$ENV{MOJO_SECRETS_FILE} = tempfile();
+
+like app->secrets->[0], qr/^[A-Za-z0-9]{44}$/, 'urandom_token generated secret';
+
+END {
+  unlink $ENV{MOJO_SECRETS_FILE};
+}
```

Comment:
Isn't the point of using `tempfile` that it gets cleaned up automatically?

Reply:
`tempfile` doesn't seem to clean up tempfiles, I've changed it to use `tempdir` instead.
  
Example with `tempfile` and the `unlink` removed:
  ```
  $ prove -Ilib  -r t/mojolicious/secret/lite-create.t -v
  t/mojolicious/secret/lite-create.t .. [2024-09-29 14:01:09.00902] [71360] [trace] Your secret passphrase has been set to strong random value and stored in "/tmp/Sp9NjlS0Es"
  [..]
  
  $ ls -l /tmp/Sp9NjlS0Es
  .rw------- sgo users 43 B Sun Sep 29 14:01:09 2024 /tmp/Sp9NjlS0Es
  ```